web/src/features/moreCast2/components/NextCastDataGrid.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import makeStyles from '@mui/styles/makeStyles'
+import { DataGrid, GridColDef, GridValueFormatterParams, GridValueGetterParams } from '@mui/x-data-grid'
+import { isNumber } from 'lodash'
+import { DateTime } from 'luxon'
+import { NextCastForecastRow } from 'features/moreCast2/interfaces'
+
+interface NextCastDataGridProps {
+  rows: NextCastForecastRow[]
+  setForecastRows: React.Dispatch<React.SetStateAction<NextCastForecastRow[]>>
+}
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexGrow: 1
+  }
+})
+
+const NextCastDataGrid = (props: NextCastDataGridProps) => {
```

Comment:
We backtracked on the "NextCast" naming once we realized it would be confusing for the users -- not sure what the best naming is for code but I use "Morecast 2" (`morecast_2`) on the backend.

Reply:
Yeah, silent protest...I liked NextCast.

I'll update.